### PR TITLE
Add reported working cameras to Reolink documentation

### LINKED ISSUE #25566: Reolink Integration
### Feedback

Hi, 

i have testet the follow cams:

works well and quickly:
RLC-520
RLC-410W

does not work
E1 Pro

### URL

https://www.home-assistant.io/integrations/reolink#reolink-firmware-limitations

### Version

2023.1.0

### Additional information

_No response_

### LINKED ISSUE #25568: Reolink Intergration
### Feedback

Hi,

I've tested the Reolink doorbell POE; it only shows camera streams.
doorbell press, lights etc. not working.

Camera which does work:
Reolink RLC-820A

### URL

https://www.home-assistant.io/integrations/reolink

### Version

2023.1.0

### Additional information

_No response_

### LINKED ISSUE #25574: Reolink Intergration
### Feedback

Hey,

The integration of my Reolink `RLC-511W` worked flawless - should be good to add it to the list of compatible products..

Thank you for the integration

### URL

https://www.home-assistant.io/integrations/reolink

### Version

2023.1.0

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Add reported cameras in #25566, #25568, #25574 to the Reolink documentation.

Additionally, sorted the list 🔠 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue:
  - fixes #25566
  - fixes #25568
  - fixes #25574

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by frenck** [comment]
@starkillerOG Could you take a look at this one?

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -34,20 +34,24 @@ The snapshots stream provides a sequence of image snapshots giving very low late
 
 The following models have been tested and confirmed to work:
 
-- RLN8-410 NVR
+- E1 Zoom
+- E1 Outdoor
+- RLC-410W
+- RLC-420-5MP
+- RLC-511W
+- RLC-520
```
Comment (starkillerOG) [inline source/_integrations/reolink.markdown:42]:
```suggestion
- RLC-520
- RLC-522
```

Comment (starkillerOG) [reply source/_integrations/reolink.markdown:42]: RLC-522 was also reported to be working here: https://github.com/home-assistant/core/issues/85420

**Review by starkillerOG — DISMISSED** [review]
Looks good to me, thanks @frenck.
Can be merged.

**Review by starkillerOG — APPROVED** [review]
Looks good to me

**Pull request merged**